src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import duke.command.*;
```

Comment:
Wild card imports `import duke.command.*` are not allowed in coding standards. To remove this auto-setting in IntelliJ, go to `Settings` > `Editor` > `Code Style` > `Java`, and set a super high number for `Class count to use import with '*'`. See bottom of the screenshot.  

![image](https://user-images.githubusercontent.com/66059643/131639728-1af1690e-acc7-476f-9e18-64052807a8a4.png)